tests/functional/s3api/test_get_object.py

```diff
@@ -85,6 +85,24 @@ def test_streaming_output_arg_with_error_response(self):
         self.assert_params_for_cmd(
             cmdline, {'Bucket': 'mybucket', 'Key': 'mykey'})
 
+    def test_invalid_expires_timestamp(self):
```

Comment:
Oh, I was under the presumption that this was mocking the response from botocore rather than the service. It was just a sanity check to make sure we don't raise an error in the case botocore does what it should in the case of an invalid Expires field in the response body from S3. Not sure the test has much value anyways, so I'll remove it for the next revisions.